src/sync-client.js

```diff
@@ -920,6 +925,8 @@ var self = {
     });
   },
 
+  getHashMethod: CryptoJS.SHA1,
```

Comment:
It's return type is defined by clients, but same problem also happens with Lawnchair.